### PR TITLE
Migrate docs from wiki to github

### PR DESCRIPTION
**Pull request by halkeye**
easier to maintain and the current standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,144 @@
-Jenkins DigitalOcean Slave Plugin
-=================================
+Allow Jenkins to start agents on [DigitalOcean](http://digitalocean.com/) droplets on demand and destroy them as they get unused.
 
-This Jenkins DigitalOcean plugin was inspired by the [ec2 plugin](https://github.com/jenkinsci/ec2-plugin) and uses the [java-digitalocean-api](https://github.com/jeevatkm/digitalocean-api-java) by [jeevatkm](https://github.com/jeevatkm) under the hood.
+# Configure plugin via Groovy script
```
Comment (timja) [inline README.md:3]: Doesn't seem like it should be a top level heading

```suggestion
## Configure plugin via Groovy script
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,144 @@
-Jenkins DigitalOcean Slave Plugin
-=================================
+Allow Jenkins to start agents on [DigitalOcean](http://digitalocean.com/) droplets on demand and destroy them as they get unused.
```
Comment (timja) [inline README.md:1]: add heading?
```suggestion
# Jenkins Digital Ocean Plugin

Allow Jenkins to start agents on [DigitalOcean](http://digitalocean.com/) droplets on demand and destroy them as they get unused.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,144 @@
...
 
-This Jenkins DigitalOcean plugin was inspired by the [ec2 plugin](https://github.com/jenkinsci/ec2-plugin) and uses the [java-digitalocean-api](https://github.com/jeevatkm/digitalocean-api-java) by [jeevatkm](https://github.com/jeevatkm) under the hood.
+# Configure plugin via Groovy script
 
-## Documentation
+Either automatically upon [Jenkins post-initialization](https://www.jenkins.io/doc/book/managing/groovy-hook-scripts/) or
+through [Jenkins script console](https://www.jenkins.io/doc/book/managing/script-console/), example:
 
-To keep a single place of truth the entire user documentation of the DigitalOcean Jenkins Slave Plugin is located in the [Jenkins Wiki Page](https://wiki.jenkins-ci.org/display/JENKINS/DigitalOcean+Plugin).
+```
```
Comment (timja) [inline README.md:8]:
```suggestion
```groovy
```

Comment (halkeye) [reply README.md:8]: oops, missed one :)

**Pull request merged**